src/index.js

```diff
@@ -115,3 +115,9 @@ export {
   Input
 } from "./components/Input";
 export { LinkList, LinkListItem } from "./components/Link";
+export { default as Modal } from "./components/Modal";
+export { default as Backdrop } from "./components/Backdrop";
+export {
+  ContainerProvider,
+  ItemContainerConsumer
```

Comment:
ItemContainerConsumer need not be exported as its used within Modal. We might also need to export BackdropProvider if its is not already. ContainerProvider and BackdropProvider will be required to be used by the component containing Modal component (as parent of Modal).

Reply:
Yea you're right exporting the consumer is unnecessary. Will update the PR